generated/model.py

```diff
@@ -81,6 +106,11 @@ class ActionRequest(BaseModel):
 
 
 class AllNotes(BaseModel):
+    """
+    Serializes all notes for a given detector, grouped by type as listed in UserProfile.NoteCategoryChoices
+    The fields must match whats in USERPROFILE.NoteCategoryChoices
```

Comment:
`USERPROFILE`

is all caps right?

Reply:
Well, no... In fact, SDK users don't have access to UserProfiles so we shouldn't even mention that here. But the source of truth for this docstring lives in the BE, so that will require a separate change